packages/client/src/views/UserAudit/UserAuditHistory.tsx

```diff
@@ -358,51 +461,52 @@ class UserAuditListComponent extends React.Component<Props, State> {
   }
 
   render() {
-    const { intl, user, theme, isLoading } = this.props
-    const { timeStart, timeEnd } = this.state
+    const { intl, user, theme, isLoading, isOnline } = this.props
+    const { timeStart, timeEnd, currentPageNumber } = this.state
     const recordCount = DEFAULT_LIST_SIZE * this.state.currentPageNumber
 
     return (
       <RecentActionsHolder id="user-audit-list">
         {isLoading && this.getLoadingView()}
         {!isLoading && (
           <>
-            <SectionTitle>
-              {intl.formatMessage(messages.auditSectionTitle)}
-            </SectionTitle>
-            <DateRangePicker
-              startDate={timeStart}
-              endDate={timeEnd}
-              onDatesChange={({ startDate, endDate }) => {
-                this.setDateRangePickerValues(startDate, endDate)
-              }}
-            />
-            <AuditListHolder>
-              <Query
-                query={FETCH_TIME_LOGGED_METRICS_FOR_PRACTITIONER}
+            <HistoryHeader>
+              <SectionTitle>
+                {intl.formatMessage(messages.auditSectionTitle)}
+              </SectionTitle>
+              <AlignedDateRangePicker
+                startDate={timeStart}
+                endDate={timeEnd}
+                onDatesChange={({ startDate, endDate }) => {
+                  this.setDateRangePickerValues(startDate, endDate)
+                }}
+              />
+            </HistoryHeader>
+            <>
+              <Query<GetUserAuditLogQuery>
+                query={GET_USER_AUDIT_LOG}
                 variables={{
-                  timeStart: timeStart.toISOString(),
-                  timeEnd: timeEnd.toISOString(),
                   practitionerId: user && user.practitionerId,
-                  locationId: user && user.locationId,
-                  count: recordCount
+                  count: recordCount,
+                  skip: DEFAULT_LIST_SIZE * (currentPageNumber - 1)
                 }}
                 fetchPolicy={'no-cache'}
               >
                 {({ data, loading, error }) => {
-                  if (error) {
+                  if (error || !data || !data.getUserAuditLog) {
                     return this.getLoadingAuditListView(true)
                   } else {
-                    const totalItems =
+                    const totalItems = Number(
                       (data &&
-                        data.fetchTimeLoggedMetricsByPractitioner &&
-                        data.fetchTimeLoggedMetricsByPractitioner.totalItems) ||
-                      0
+                        data.getUserAuditLog &&
+                        data.getUserAuditLog.total) ||
+                        0
+                    )
 
                     return (
                       <Table
```

Comment:
>Refactor the query results to use the **TableView** component. Use Pagination component like we do in the History component of RecordAudit [(reference)](https://github.com/opencrvs/opencrvs-core/blob/component-library-v2/packages/client/src/views/RecordAudit/History.tsx#L309). - 2

https://github.com/opencrvs/opencrvs-core/issues/3640

Reply:
This was the thing @jpye-finch also mentioned. Also make sure the column sizes match the design

Reply:
The new component library has  `Table ` and not  `TableView`. This component `Table` comes with pagination and that's what is been used in the project currently
<img width="386" alt="table" src="https://user-images.githubusercontent.com/66411673/193064913-4dc18bc7-5623-4fe3-ad69-d3234fae73a8.png">

Reply:
Yes check this example 
https://ui-kit.farajaland-staging.opencrvs.org/?path=/docs/typography-text--heading-with-icon

Reply:
yes UserAuditHistory.tsx is using the latest table component <Table />  
You can set the column widths. Make sure they add up to 100. So no scrollbar is required.

https://ui-kit.farajaland-staging.opencrvs.org/?path=/docs/data-table--pagination-footer-and-sorting

Please also check and fix the sort icon.